deltachat-ios/Handler/ProgressAlertHandler.swift

```diff
@@ -2,34 +2,46 @@ import UIKit
 import DcCore
 
 protocol ProgressAlertHandler: UIViewController {
-    var progressAlert: UIAlertController { get }
-    var configureProgressObserver: Any? { get set }
-    func showProgressAlert(title: String)
+    var progressAlert: UIAlertController? { get set }   // needs to be implemented as weak
```

Comment:
please add the the comment _why_ this is needed. at least to me this is not obvious.